bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/regexp/TreeBuilder.java

```diff
@@ -182,6 +243,11 @@ private String readRegUnicodeScript() {
         throw new BallerinaException(getErrorMsg(nextToken));
     }
 
+    /**
+     * Read the unicode general category.
+     *
+     * @return gc=category or category
```

Comment:
```suggestion
     * @return gc=category
```

Reply:
or else
`gc=general category`

Reply:
Refer [#commet](https://github.com/ballerina-platform/ballerina-lang/pull/40003#discussion_r1193165349)

Reply:
I think both me and @sanjana are suggesting to make that more clear.

Reply:
Fixed.